x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/sourcerer/sourcerer_timeline.cy.ts

```diff
@@ -92,24 +99,23 @@ describe('Timeline scope', { tags: ['@ess', '@serverless', '@brokenInServerless'
     });
   });
   describe('Alerts checkbox', () => {
-    before(() => {
+    beforeEach(() => {
       login();
+      deleteTimelines();
       createTimeline(getTimeline()).then((response) =>
         cy.wrap(response.body.data.persistTimeline.timeline.savedObjectId).as('timelineId')
       );
       createTimeline(getTimelineModifiedSourcerer()).then((response) =>
         cy.wrap(response.body.data.persistTimeline.timeline.savedObjectId).as('auditbeatTimelineId')
       );
-    });
-
-    beforeEach(() => {
-      login();
       visitWithTimeRange(TIMELINES_URL);
       refreshUntilAlertsIndexExists();
     });
 
     it('Modifies timeline to alerts only, and switches to different saved timeline without issue', function () {
-      openTimelineById(this.timelineId).then(() => {
+      openTimelineFromSettings();
+      cy.get(OPEN_TIMELINE_MODAL).should('be.visible');
```

Comment:
If this assertion is always proceeding `openTimelineFromSettings`, should it not be moved into there? Is there a situation where it wouldn't be needed? I see we have assertions in other such task helpers, and `openTimelineFromSettings` had the assertion there before (albeit _before_ the click, not after 🤷 ); can we just fix the order of those calls within `openTimelineFromSettings` ?